### ISSUE TITLE
Fix tabbing in wishlist

### ISSUE THREAD

**Issue by marlass** [SAP/cloud-commerce-spartacus-storefront#5937]
Make img not focusable with `[tabindex]`

Add new tabbing order test for that feature.
